index.html

```diff
@@ -29,15 +29,18 @@
     <!-- CORE JQUERY SCRIPTS -->
     <!-- BOOTSTRAP SCRIPTS  -->
     <!-- <script src="assets/js/bootstrap.js"></script> -->
+
```

Comment:
Is there any specific reason behind adding extra blank lines here and other two places